includes/razorpay-subscriptions.php

```diff
@@ -281,7 +367,7 @@ public function getDisplayAmount($order)
 
         $sub = $this->getWooCommerceSubscriptionFromOrderId($order->get_id());
 
-        $recurringFee = $sub->get_amount();
+        $recurringFee = $sub->get_total();
```

Comment:
Looks correct, but can you clarify what's the difference.

Reply:
@captn3m0 `get_amount` doesn't exist in the `WC_Subscription` class. This is a bug.